enhancements/20220525-system-logging.md

```diff
@@ -0,0 +1,174 @@
+# System Logging
+
+## Summary
+
+We need to be able to support exporting harvester system logs outside the cluster, to allow for collecting and analyzing those logs to aid in debugging and harvester management.
+
+### Related Issues
+
+- https://github.com/harvester/harvester/issues/577
+- https://github.com/harvester/harvester/issues/2644
+- https://github.com/harvester/harvester/issues/2645
+- https://github.com/harvester/harvester/issues/2646
+- https://github.com/harvester/harvester/issues/2647
+
+## Motivation
+
+### Goals
+
+List the specific goals of the enhancement. How will we know that this has succeeded?
+
+- [X] The user can aggregate harvester logs in a centralized location
+  - [X] k8s cluster logs
+  - [X] host system logs (ie `/var/log`)
+- [X] The user can export the aggregated harvester logs outside the cluster (ex rancher)
+
+### Non-goals
+
+- this enhancement does not cover integration into the harvester / rancher UI (but ideally this will eventually be implemented as well)
+
+## Proposal
+
+We will deploy a new `ManagedChart` in the [Harvester Installer](https://github.com/harvester/harvester-installer) to
+install. The `ManagedChart` will deploy a `ClusterFlow` to select and aggregate interesting logs, which can be managed
+by a new `ClusterOutput` crd.
+
+To collect the host system logs, a new harvester sub-chart [harvester-journal](https://github.com/joshmeranda/harvester-charts/tree/logging/charts/harvester-journal).
+
+### User Stories
```

Comment:
Please add a story, `enable logging by default`

The `logging` is enabled in v1.1.0 by default. Disable logging is not supported in v1.1.0.

It has `fluentbit` as agent, `fluentd` as center. `fluentd` is not backed by PVC, users need to set receiver accordingly to  receive log on time.